docs/datasets/kitti_det.md

```diff
@@ -0,0 +1,230 @@
+# KITTI Dataset
+
+This page provides specific tutorials about the usage of MMDetection3D for kitti dataset.
```

Comment:
Remind the user that the current tutorial is only for LiDAR-based and multi-modality 3D detection methods. Contents related to monocular methods will be supplemented afterward.